docs/proposals/hosted_components_templates.md

```diff
@@ -0,0 +1,559 @@
+# Use [OCP templates and] ABPs to install hosted components
+
+## Description
+Currently most of our hosted components have their own repositories where they
+maintain changes and then need to synchronize those changes within their
+respective roles. This can cause cause delays and times where features don't
+match up with what is installed or can be configured.
```

Comment:
This needs feedback from teams that manage components - @deads2k, @dgoodwin, @bparees and a few others for initial comment.